webhookValidationConfig.go

```diff
@@ -37,6 +39,16 @@ type TTLVConfig struct {
 	Now    func() time.Time
 }
 
+type OptionsConfig struct {
```

Comment:
does it make sense to move this from webhook-schema to ancla? List is derived from the options provided by webhook-schema